Csharprepl cannot find script in current working directory

### Version

0.6.5

### What happened?

If I create a simple hello world CSX script file in my current working directory and then try to run it using `csharprepl test.csx` in my Powershell terminal I get the following output:

```
Welcome to the C# REPL (Read Eval Print Loop)!
Type C# expressions and statements at the prompt and press Enter to evaluate them.
Type help to learn more, exit to quit, and clear to clear your terminal.

Running supplied CSX file...
┌────────────────────────────────────────Exception────────────────────────────────────────┐
│ (1,7): error CS1504: Source file 'test.csx' could not be opened -- Could not find file. │
└─────────────────────────────────────────────────────────────────────────────────────────┘
```

The weird thing is that when I do `Directory.GetCurrentDirectory()` I indeed get back my current working directory.

To actually get it to work I need to run

`csharprepl (ls test.csx | select -expandproperty fullname)` 

To retrieve the full file name, which is not ideal.

--- a/CSharpRepl.Services/Roslyn/Scripting/ScriptRunner.cs
+++ b/CSharpRepl.Services/Roslyn/Scripting/ScriptRunner.cs
@@ -61,6 +61,7 @@ internal sealed class ScriptRunner
         );
         this.scriptOptions = ScriptOptions.Default
             .WithMetadataResolver(metadataResolver)
+            .WithSourceResolver(compilationOptions.SourceReferenceResolver)
             .WithReferences(referenceAssemblyService.LoadedImplementationAssemblies)
             .WithAllowUnsafe(compilationOptions.AllowUnsafe)
             .WithLanguageVersion(LanguageVersion.Preview)

--- a/CSharpRepl.Tests/ReadEvalPrintLoopTests.cs
+++ b/CSharpRepl.Tests/ReadEvalPrintLoopTests.cs
@@ -120,6 +120,24 @@ public class ReadEvalPrintLoopTests : IClassFixture<RoslynServicesFixture>
     }
 
     [Fact]
+    public async Task RunAsync_LoadScriptByFilePath_RunsScript()
+    {
+        prompt
+            .ReadLineAsync()
+            .Returns(
+                new PromptResult(true, "exit", default)
+            );
+
+        await repl.RunAsync(new Configuration(
+            loadScript: "#load \"Data\\LoadScript.csx\""
+        ));
+
+        Assert.DoesNotContain("Exception", console.AnsiConsole.Output);
+        Assert.DoesNotContain("CS1504", console.AnsiConsole.Output);
+        Assert.DoesNotContain("Could not find file", console.AnsiConsole.Output);
+    }
+
+    [Fact]
     public async Task RunAsync_Reference_AddsReference()
     {
         prompt

--- a/CSharpRepl.Tests/ReadEvalPrintLoopTests.cs
+++ b/CSharpRepl.Tests/ReadEvalPrintLoopTests.cs
@@ -129,7 +129,7 @@ public class ReadEvalPrintLoopTests : IClassFixture<RoslynServicesFixture>
             );
 
         await repl.RunAsync(new Configuration(
-            loadScript: "#load \"Data\\LoadScript.csx\""
+            loadScript: "#load \"Data/LoadScript.csx\""
         ));
 
         Assert.DoesNotContain("Exception", console.AnsiConsole.Output);

